ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dhcp-server.j2

```diff
@@ -10,16 +10,65 @@
 {%         do ethernet_interfaces_dhcp_server.append(ethernet_interface) %}
 {%     endif %}
 {% endfor %}
-{# later on this condition will change when top level key dhcp_server is implemented #}
-{% if ethernet_interfaces_dhcp_server | length > 0 %}
+{% if (ethernet_interfaces_dhcp_server | length > 0 or dhcp_servers is arista.avd.defined) %}
 
 ## DHCP Server
+{%     if dhcp_servers is arista.avd.defined %}
+
+### DHCP Servers Summary
+
+| DHCP Server Enabled | VRF | IPv4 DNS Domain | IPv6 DNS Domain |
+| ------------------- | --- | --------------- | --------------- |
+{%         for dhcp_server in dhcp_servers | arista.avd.natural_sort %}
```

Comment:
Should you not sort the same way as in config template using the `vrf` key ? or maybe need to sort twice, once per VRF and once per name?

Reply:
There's no name in the data model for the server itself, only for the subnet.